### ISSUE TITLE
Add AArch64 Linux support

### ISSUE THREAD

**Issue by robsmith11** [TuringLang/Libtask.jl#32]
Idk how difficult it is, but I see there is already support for 32bit arm.  Is it possible to add the 64bit libraries as well?

```
julia> versioninfo()
Julia Version 1.3.0-DEV.0
Platform Info:
  OS: Linux (aarch64-unknown-linux-gnu)
  CPU: unknown
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, kryo)
```

**Comment by yebai** [comment]
This should be straightforward once there is an official Julia `AArch64` binary.

**Comment by robsmith11** [comment]
Great! AArch64 was recently readded to the download page for the latest release:

https://julialang.org/downloads/

**Comment by yebai** [comment]
@robsmith11 This should be fixed by #33. Please give it a try and let us know if you have any further problems.

**Comment by robsmith11** [comment]
Thanks!  It works for me.
